scripts/templates/soccer.wbt.template

```diff
@@ -1942,5 +2006,4 @@ Robot {
   ]
   controller "rcj_soccer_referee_supervisor"
   supervisor TRUE
-  synchronization TRUE
```

Comment:
@Adman does this mean `synchronization` is set to `TRUE` by default? Looking at your commit (https://github.com/RoboCupJuniorTC/rcj-soccer-sim/commit/2dbabb77aaa1c4c7452c650d9d51eab387ce5bbb), it does sound like something we'd like to keep set to `TRUE` for the foreseeable future, right?

Reply:
@mrshu Yeah synchronization is set to `TRUE` by default, that's why webots deletes the property everytime the world is saved. I reverted it back so that it is explicitly set to `TRUE`. Thanks for pointing out.